services/api/src/main/java/io/swec/services/api/Main.java

```diff
@@ -20,6 +20,10 @@
     // Base URI the Grizzly HTTP server will listen on
     public static final String BASE_URI = "http://0.0.0.0:9081/myapp/";
 
+    // Sonar issue
+    // String uname = "steve";
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://159.65.230.204:9000/coding_rules#rule_key=squid%3ACommentedOutCodeLine)